torchvision/transforms/transforms.py

```diff
@@ -543,7 +543,13 @@ class RandomResizedCrop(object):
     """
 
     def __init__(self, size, scale=(0.08, 1.0), ratio=(3. / 4., 4. / 3.), interpolation=Image.BILINEAR):
-        self.size = (size, size)
+        if isinstance(size, tuple):
+            self.size = size
+        else:
+            self.size = (size, size)
+        if (scale[0] > scale[1]) or (ratio[0] > ratio[1]):
+            raise ValueError("inappropriate range(min,max) for scale or ratio supplied")
```

Comment:
This is not necessary since random.uniform doesn't limit upper/lower bound order.

Reply:
@kouyoumin  this was to prevent sneaky errors like somebody wanted to put (0.8,1.7) but accidently put (0.8,0.7).
Instead of propagating wrong results throughout the pipeline .
IMO it is better to raise an error.
@kouyoumin  what do you say ?

Reply:
Such typo is hard to eliminate. For example, (0.6, 1.7) -> (0.6, 0.7) will pass.
I think it's too much to raise an error, maybe a warning message is enough?

Reply:
sounds cool to me !! Warning should be enough @kouyoumin